scripts/create-project-structure.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/sh
+# -----------------------------------------------------------------------------
+# Script to create a standard project directory structure and initial markdown files.
+# Usage: ./create-project-structure.sh <project>
+#   <project> - The name of the project to create directories and files for.
+# Example:
+#   ./create-project-structure.sh my-new-project
+# -----------------------------------------------------------------------------
+cd "$(dirname "$(readlink -f "$0")")/.."
+project="$1"
+if [ -z "$project" ]; then
+    echo "Usage: $0 <project>"
+    exit 1
+fi
+start_ms=$(date +%s%3N)
+# Create directories
+mkdir -p "./src/${project}/about-the-device" \
+         "./src/${project}/getting-started" \
+         "./src/${project}/faq-about-the-product" \
+         "./src/${project}/resources" \
+         "./src/${project}/troubleshooting"
+echo "---
+zendesk:
+  category_id:
+  position: 99
+  name: ${project}
+  description: ${project}
+---
+" > "./src/${project}/_category.md"
+echo "---
+zendesk:
+  section_id:
+  position: 20
+  name: About the device
+---
+" > "./src/${project}/about-the-device/_section.md"
+echo "---
+zendesk:
+  section_id:
+  position: 50
+  name: FAQ - About the product
+---
+" > "./src/${project}/faq-about-the-product/_section.md"
+echo "---
+zendesk:
+  section_id:
+  position: 60
+  name: Resources
+---
+" > "./src/${project}/resources/_section.md"
+echo "---
+zendesk:
+  section_id:
+  position: 10
+  name: Getting started
+---
+" > "./src/${project}/getting-started/_section.md"
+echo "---
+zendesk:
+  section_id:
+  position: 40
+  name: Troubleshooting
+---
+" > "./src/${project}/troubleshooting/_section.md"
+echo "---
+---
+zendesk:
+  article_id:
+  name: Datasheet
+  position: 40
+  labels: resources
+---
+" > "./src/${project}/resources/datasheet.md"
+echo "---
+zendesk:
+  article_id:
+  name: Declaration of Conformity for EU
+  position: 30
+  labels: resources
+---
+" > "./src/${project}/resources/eu-declaration-of-conformity.md"
```

Comment:
It would be nice if there was some order here.
So the articles would be done right after the sections for dem.
So like:
"./src/${project}/resources/_section.md"
"./src/${project}/resources/eu-declaration-of-conformity.md"

And so on.
Now it is:
"./src/${project}/resources/_section.md" <-- creating the section
"./src/${project}/getting-started/_section.md" <-- creating another section
"./src/${project}/troubleshooting/_section.md" <-- creating another section
"./src/${project}/resources/datasheet.md" <-- creating content for the resources section.


Or better separate the parts.